interface/main/left_nav.php

```diff
@@ -1841,12 +1838,7 @@ function selpopup(selobj) {
         <?php $myrow = sqlQuery("SELECT state FROM registry WHERE directory = 'track_anything'");
         if ($myrow['state']=='1') {
             genTreeLink('RTop', 'tan', xl('Configure Tracks'));
-        } ?>
-        <?php if (!$GLOBALS['use_active_directory']) {
-            genTreeLink('RTop', 'pwd', xl('Password'));
-        } ?>
-        <?php genTreeLink('RTop', 'mfa', xl('MFA Manaagement')); ?>
-        <?php genMiscLink('RTop', 'prf', '0', xl('Preferences'), 'super/edit_globals.php?mode=user'); ?>
+        } ?>       
```

Comment:
Recommend reverting all the changes in left_nav.php since the Frames gui does not have access to the top right Users menu (note the Frames gui option is going to be removed for future 5.0.3, so this entire script will be disappearing :) ). Otherwise this PR looks good.

Reply:
@bradymiller Will left nav menu work if I leave it there? Looks like I have to revert the Custom.json as well.

Reply:
Hi @dimitristsilis,

The left nav is specific to the "frames gui" and the custom.json is specific to the "tabs gui":
https://www.open-emr.org/wiki/index.php/Administration_Globals#Layout_.28need_to_logout.2Flogin_after_change_this_setting.29

I'd rec not touching left_nav and just focusing on the json changes.

Reply:
@bradymiller Thanks, was curious why custom.json is not inside tabs folder.

Reply:
@dimitristsilis ,

This is because the Custom.json is meant to be configurable (and can add other menus in that directory), so is stored in a writable directory:
https://www.open-emr.org/wiki/index.php/Role_Based_Menus

The hope is that someday somebody makes a menu editor gui for that stuff :)